.github/workflows/test.yml

```diff
@@ -101,20 +101,14 @@ jobs:
           mobify_user: ${{ secrets.MOBIFY_CLIENT_USER }}
           mobify_api_key: ${{ secrets.MOBIFY_CLIENT_API_KEY }}
 
+      # TODO - Do we still need this?
       - name: Push Bundle to MRT (Development)
         if: env.IS_NOT_FORK == 'true' && env.IS_MRT_NODE == 'true' && env.DEVELOP == 'true'
         uses: "./.github/actions/push_to_mrt"
         with:
           CWD: "./packages/template-retail-react-app"
           TARGET: staging
 
-      - name: Push Bundle to MRT (Production)
-        if: env.IS_NOT_FORK == 'true' && env.IS_MRT_NODE == 'true' && env.RELEASE == 'true'
-        uses: "./.github/actions/push_to_mrt"
-        with:
-          CWD: "./packages/template-retail-react-app"
-          TARGET: production
```

Comment:
Removing this since this is redundant. We are now deploying a preset to the demo site when a new latest release tag is produced.

This does mean that we won't be deploying preview releases onto the live demo site but I can create a different step to handle this if we want.